Fix #2556, flight like platformed defined values

dmknutsen opened the pull request:
**Checklist (Please check before submitting)**

* [x] I reviewed the [Contributing Guide](https://github.com/nasa/cFE/blob/main/CONTRIBUTING.md).
* [x] I signed and emailed the appropriate [Contributor License Agreement](https://github.com/nasa/cFS/blob/main/CONTRIBUTING.md#contributor-license-agreement-cla) to GSFC-SoftwareRelease@mail.nasa.gov and copied cfs-program@lists.nasa.gov.

**Describe the contribution**
Updates CFE_PLATFORM_EVS_LOG_MAX  and CFE_PLATFORM_EVS_APP_EVENTS_PER_SEC to align with 'flight like' values 
- Fixes #2556 

**Testing performed**
CI pipeline
BVT testing

**System(s) tested on**
 - Ubuntu 20.04
 - cFE Development Build equuleus-rc1+dev131 (Codename Equuleus), Last Official Release: cFE v6.7.0)

**Contributor Info - All information REQUIRED for consideration of pull request**
Dan Knutsen
NASA Goddard

skliper commented:
Might be a surprise to anyone using the defaults?  Is there a way to match the BVTs w/o open source behavior changes?

dmknutsen commented:
Yep - we could just put the updates in the example_platform_cfg.h.  I'm a bit surprised that the functional tests failed, as I just changed CFE_PLATFORM_EVS_APP_EVENTS_PER_SEC back to it's original value that I was thinking was more flight like.  The only reason we changed it in the first place was to prevent test results from being squelched in the console output (cfe_test.log has always been correct).  In any case, just using example_platform_cfg.h fixes both issues...I'll push an update.

dzbaker commented:
@dmknutsen Would you be able to go over this one at next week's CCB?
